releases/release-1.30/release-notes/maps/pr-120159-map.yaml

```diff
@@ -1,3 +1,3 @@
 pr: 120159
 releasenote:
-  text: Fixed delete pod declare no controllor note.
+  text: Fixed "delete pod declare no controllor" note.
```

Comment:
```suggestion
  text: Modified the error message of `unmanagedFatal` to enhance clarity while preserving grammatical consistency with `unmanagedWarning`. This improvement ensures a more understandable prompt for users.
```

From the PR -

Reply:
Good job Faeka :+1:  I was expecting this.